velox/expression/tests/ExpressionRunnerTest.cpp

```diff
@@ -52,6 +53,12 @@ DEFINE_string(
     "flag are mutually exclusive. If both are specified, --sql is used and "
     "--sql_path is ignored.");
 
+DEFINE_string(
+    registry,
+    "presto",
+    "Funciton registry to use for expression evaluation. Currently supported values are "
```

Comment:
Sorry, one small nit, "Function" is misspelled here.

Reply:
Thank you! Great catch. Changing now

Reply:
Do you have access to the Linter failure? Any idea why it is failing?